mini-e2e-helm.groovy

```diff
@@ -176,7 +176,7 @@ node('cico-workspace') {
 		}
 		stage('run e2e') {
 			timeout(time: 120, unit: 'MINUTES') {
-				ssh "cd /opt/build/go/src/github.com/ceph/ceph-csi && make run-e2e NAMESPACE='${namespace}' E2E_ARGS='--deploy-cephfs=false --deploy-rbd=false'"
+				ssh "cd /opt/build/go/src/github.com/ceph/ceph-csi && make run-e2e NAMESPACE='${namespace}' E2E_ARGS='--deploy-cephfs=false --deploy-rbd=false --helm-test=true'"
```

Comment:
if this option does not get handled by the e2e tests, would the test still run, or does it fail immediately?

Reply:
As per my testing locally, any new parameter(like `--helm-test=true` in this case)  is just ignored and the execution continues normally